### DISCUSSION TITLE
No Wekan Integration Section

### DISCUSSION THREAD

**Discussion by CaptainPackers** [kromitgmbh/titra#54] [Q&A]
I have no Wekan Integration section or checkbox on my local docker installation of titra. About page says I am running version 0.44.1. Only options are

- titra API token
- Siwapp URL
- Siwapp API token
- URL of the Zammad instance
- Zammad Personal Access Token

Any help would be appreciated.

**Comment by faburem** [comment]
This should be fixed in the latest version of titra. Please report back if that is not the case!

**Reply by CaptainPackers** [reply]
All set. Thanks.
